src/state/image-dataset/csv-dataset/index.ts

```diff
@@ -0,0 +1,398 @@
+import { ViewerChannelSettings } from "@aics/web-3d-viewer";
+import { Album } from "../..";
+import * as Papa from "papaparse";
+import {
+    DataForPlot,
+    MeasuredFeatureDef,
+    FileInfo,
+    PerCellLabels,
+    MeasuredFeaturesOption,
+    DiscreteMeasuredFeatureDef,
+    ContinuousMeasuredFeatureDef,
+} from "../../metadata/types";
+import { ImageDataset, InitialDatasetSelections, Megaset } from "../types";
+import firebase from "firebase";
+import {
+    CELL_ID_KEY,
+    FOV_ID_KEY,
+    FOV_THUMBNAIL_PATH,
+    FOV_VOLUME_VIEWER_PATH,
+    GROUP_BY_KEY,
+    THUMBNAIL_PATH,
+    TRANSFORM,
+    VOLUME_VIEWER_PATH,
+} from "../../../constants";
+
+export const DEFAULT_CSV_DATASET_KEY = "csv";
+
+const METADATA_KEYS = new Set([
+    CELL_ID_KEY,
+    FOV_ID_KEY,
+    FOV_THUMBNAIL_PATH,
+    FOV_VOLUME_VIEWER_PATH,
+    THUMBNAIL_PATH,
+    VOLUME_VIEWER_PATH,
+    TRANSFORM,
+]);
+
+// From Adobe categorical colors
+const DEFAULT_COLORS = [
+    "#27B4AE",
```

Comment:
Without context, I don't get this or why they are in this order.  If it makes sense to people with context, great.